sklearn/utils/estimator_checks.py

```diff
@@ -684,7 +686,11 @@ def check_fit_score_takes_y(name, Estimator):
 @ignore_warnings
 def check_estimators_dtypes(name, Estimator):
     rnd = np.random.RandomState(0)
-    X_train_32 = 3 * rnd.uniform(size=(20, 5)).astype(np.float32)
+    if name in ["BoxCoxTransformer", "SpectralCoclustering",
```

Comment:
Sorry this was my hack for making tests pass on my machine. There is one more PR where common tests are being enabled for biclustering module and these modifications for that changes. Will remove them here.